sdk/src/types/block/output/basic.rs

```diff
@@ -373,26 +369,27 @@ pub(crate) mod dto {
         #[serde(rename = "type")]
         pub kind: u8,
         // Amount of IOTA tokens held by the output.
-        pub amount: String,
+        #[serde(with = "string")]
```

Comment:
lets stick with 1 or the other way to import this. i prefer the full qualifier

Reply:
I prefer an import

Reply:
Then update the one below please, it has a full import